README.rst

```diff
@@ -134,6 +134,11 @@ ACME setup
 For details on having Synapse manage your federation TLS certificates
 automatically, please see `<docs/ACME.md>`_.
 
+.well-known setup
```

Comment:
I'd prefer it if we could avoid adding more top-level headings to the README, which is already pretty complicated.